commons/connector/altoida/AltoidaLog.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "radarcns.connector.altoida",
+  "type": "record",
+  "name": "AltoidaLog",
+  "doc": "Logs of actions taken.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "Timestamp in unix time acquired locally from the device."},
+    {"name": "timeRecieved", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+    {"name": "log_actions", "type": "string", "doc": "Log of actions taken during test."},
+    {"name": "extra_info_actions", "type": "string", "doc": "Extra information about action taken."}
```

Comment:
What kind of info would this be? Just `extraInfo` may be better. If any of the following is a better fit, please use that: `description`, `notes` or `comments`